bucket/scene-builder.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "15.0.1",
+    "description": "A visual, drag n drop, layout tool for designing JavaFX application user interfaces.",
+    "homepage": "https://gluonhq.com/products/scene-builder/",
+    "license": "BSD-3-Clause",
+    "url": "https://download2.gluonhq.com/scenebuilder/15.0.1/install/windows/SceneBuilder-15.0.1.msi",
```

Comment:
![A](https://i.imgur.com/G0d8euc.png)

Reply:
I am a bit confused.

What's the issue?

Reply:
64bit only...